classes/plugins/load.py

```diff
@@ -0,0 +1,75 @@
+from multiprocessing import Array
+from . import api
+from . import event
+
+import os
+import logging
+import inspect
+
+def _getPlugins_() -> Array:
+    import pkgutil
+    import importlib
+    if not (os.path.exists('plugins/')):
+        os.mkdir('plugins/')
+    import plugins as exts
+
+    def unqualify(name: str) -> str:
+        """Return an unqualified name given a qualified module/package `name`."""
+        return name.rsplit(".", maxsplit=1)[-1]
+    plugins = {}
+    for module in pkgutil.walk_packages(exts.__path__, f"{exts.__name__}."):
+        plugin_name = unqualify(module.name)
+        if plugin_name.startswith("_"):
+            next
+        plugins[plugin_name] = importlib.import_module(module.name)
+    order = {}
+    for plugin_name in plugins:
+        plugin = plugins[plugin_name]
+        try:
+            tmp = order[int(plugin.PRIORITY)]
+        except KeyError:
+            order[int(plugin.PRIORITY)] = []
```

Comment:
Check out `collections.defaultdict`, it's a dictionary whose keys are all some implicit type, and avoids you having to check whether it exists/what type it is/etc

Reply:
done